### ISSUE TITLE
Tween didn't repeat anymore after being stopped  then started again.

### ISSUE THREAD

**Issue by jmb462** [godotengine/godot#40679]
**Godot version:**
This bug was introduced in 3.2.3 RC1
Probably with this commit : 8ef40b9 ( "bug with Tween.is_active, fixes #39760" )

**OS/device including version:**
Windows and Mac. Not tested under Linux OS

**Issue description:**
When a tween is stopped using Tween.stop() and then started again with Tween.start(), the tween never repeat even if Tween.repeat is true.

**Steps to reproduce:**
Configure a tween with repeat true.
Start it (tween is looping normaly)
Stop it.
Start it again (tween never repeat even if Tween.repeat is still true.

**Minimal reproduction project:**
Here is a minimal projet :
[tween_bug.zip](https://github.com/godotengine/godot/files/4974081/tween_bug.zip)

**Comment by fabriceci** [comment]
The bug's fix https://github.com/godotengine/godot/commit/8ef40b930641baa0206528ed7461334c5227758a introduce a vicious bug on my game. In some case when I launch a Tween, it interrupt and emits a completed signal just 3ms after the launch. Unfortunately, I was unable to reproduce the issue in a new project.

**Comment by akien-mga** [comment]
I reverted the cherry-pick of #39801 with 2d42625184cd70e954fe6ef8dc6c95a23e13b9e2, which should fix this issue for 3.2.3 (reintroducing #39760).

#39801 is not reverted in the `master` branch so the issue is still valid there, we can look for a bugfix first and then decide if we can fix both #39760 and the present issue, or if a revert is also the better option there.

**Comment by jmb462** [comment]
To complete fabriceci comment, this new problem happens when Tween.remove_all() is called when the tween is not yet active.
It prevent the next tween to be processed and finished_all signal is directly emitted when tween is launched.
I don't know if it deserve another issue because 3.2.3 and 4.0 are not synchronised anymore about the tweens issues.

Here is a minimal project for the bug that fabricci is talking about.

[bug_tween_3.zip](https://github.com/godotengine/godot/files/5123963/bug_tween_3.zip)

**Comment by FarLanderCraft** [comment]
I'm still having this issue.
I last updated Godot Engine on May 7, 2021, and I'm having this issue with tweens.
```
if Input.is_action_just_pressed("input"):
    if booleanValue:
      tween.interpolate_property($Sprite, "self_modulate", Color("ffffff"), Color("b72424"), 1, Tween.TRANS_SINE, Tween.EASE_IN_OUT)
      tween.start
    else:
      tween.stop($Sprite, "self_modulate")
```
It repeats correctly the first time, but every other time after it stops it doesn't loop. ;-;

**Comment by jmb462** [comment]
A PR is already merged on master branch and a PR for 3.x branch is done but not merged yet because @akien-mga said it was submitted too late before the 3.3 release and it was risky to merge at that time. #47142

**Comment by nightblade9** [comment]
I just ran into this and it's quite problematic for me in 3.3.3. (Stopping and starting the tween, doesn't start it.)

1) Is this still slated to be fixed in 3.4?
2) Is there any work-around for this right now?

**Comment by jmb462** [comment]
The fix is already merged in 3.x branch but still not cherrypicked in 3.3
